includes/Services/FlowService.php

```diff
@@ -0,0 +1,191 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+class FlowService {
+
+	// An array to keep the Flow Data updated in the database
+	protected $updated_flow_data = array();
+
+	public static function initalize_flow_data() { 
+		$default_flow_data = Flows::get_data();
+        if ( ! ( $flow_data = self::read_flow_data_from_wp_option() ) )
+            return self::get_default_flow_data();
+		$updated_flow_data = self::update_flow_data_recursive($default_flow_data, $flow_data, $updated_flow_data);
+		self::update_wp_options_data_in_database($updated_flow_data);
+        return $updated_flow_data;
+    }
+
+    private static function get_default_flow_data() {
```

Comment:
```suggestion
    public static function get_default_flow_data() {
```
No use making it private cause this can either ways be read via the other public functions.